terraform/deployments/govuk-test/outputs.tf

```diff
@@ -2,26 +2,6 @@ output "private_subnets" {
   value = data.terraform_remote_state.infra_networking.outputs.private_subnet_ids
 }
 
-output "publisher-web_security_groups" {
-  value = module.govuk.publisher_security_groups
-}
-
-output "frontend_security_groups" {
-  value = module.govuk.frontend_security_groups
-}
-
-output "signon_security_groups" {
-  value = module.govuk.signon_security_groups
-}
-
-output "content-store_security_groups" {
-  value = module.govuk.content_store_security_groups
-}
-
-output "draft-content-store_security_groups" {
-  value = module.govuk.draft_content_store_security_groups
-}
```

Comment:
OK to remove these for now, but we are likely to need them or an equivalent later on.

The reason for these is that unscheduled tasks in ECS must have the same security groups as the ECS Service they would be associated with (e.g. for db migrations, but also for ad-hoc rake tasks like `tools/run-task.sh`).